client/app/validations/submittable-pas-form.js

```diff
@@ -62,4 +63,130 @@ export default {
       message: 'This field is required',
     }),
   ],
+  dcpZoningauthorizationpursuantto: [
+    validatePresenceIf({
+      presence: true,
+      comparison: 'not',
+      on: 'dcpPfzoningauthorization',
+      withValue: null || undefined,
```

Comment:
I think I see what you're doing but this will evaluate immediately when it's defined: 
![image](https://user-images.githubusercontent.com/5004319/82274121-54a74000-994d-11ea-8b0c-adaa6018c8bf.png)

So I don't think it does what you think exactly.

The trick is it'd have to be a function:

```
withValue(something) {
  return something === null || something === undefined;
}
```

The validatePresenceIf code would have to be updated to do this accordingly.